extensions/sql-database-projects/src/test/dialogs/publishDatabaseDialog.test.ts

```diff
@@ -7,20 +7,20 @@ import * as should from 'should';
 import * as path from 'path';
 import * as os from 'os';
 import * as vscode from 'vscode';
-import * as baselines from './baselines/baselines';
-import * as templates from '../templates/templates';
-import * as testUtils from './testUtils';
+import * as baselines from '../baselines/baselines';
```

Comment:
the changes in this file are from moving the dialog tests into their own folder in the tests folder